2018-09-13 20:55:42  [ main:3401 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@3f6f67012018-09-13 20:55:42  [ main:3414 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@534a5a982018-09-13 20:55:42  [ main:3452 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@130c12b72018-09-13 20:55:42  [ main:3516 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@5d534f5d2018-09-13 20:55:42  [ main:3550 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@4d5650ae2018-09-13 20:55:42  [ main:3588 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@51c668e32018-09-13 20:55:42  [ main:3603 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@4a883b152018-09-13 20:58:48  [ main:3502 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@3f6f67012018-09-13 20:58:48  [ main:3514 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@534a5a982018-09-13 20:58:48  [ main:3536 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@130c12b72018-09-13 20:58:48  [ main:3596 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@5d534f5d2018-09-13 20:58:48  [ main:3626 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@4d5650ae2018-09-13 20:58:48  [ main:3668 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@51c668e32018-09-13 20:58:48  [ main:3683 ] - [ ERROR ]  生成表格的过程中出现了未知错误,org.apache.poi.openxml4j.exceptions.OpenXML4JRuntimeException: Fail to save: an error occurs while saving the package : The part /docProps/app.xml failed to be saved in the stream with marshaller org.apache.poi.openxml4j.opc.internal.marshallers.DefaultMarshaller@4a883b15